docs/processing.md

```diff
@@ -140,6 +140,8 @@ contrib components, and in the future can even allow user plugins possibly throu
 [HTTP proxies](https://github.com/proxy-wasm/spec). The arguments to operations will primarily be field expressions,
 allowing the operation to mutate telemetry as needed.
 
+There are times when the query language input and the underlying telemetry model do not translate cleanly.  For example, a span ID is represented in pdata as a SpanID struct, but in the query language it is more natural to represent the span ID as a string or a byte array.  The solution to this problem is Constructors. Constructors are functions that help translate between the query language input into the underlying pdata structure.  These types of functions do not change the telemetry in any way.  Instead, they manipulate the query language input into a form that will make working with the telemetry easier or more efficient.
```

Comment:
I'd find "Factory" to be more familiar than "Constructor" for this sort of concept, but maybe just me

Reply:
Factory is good with me too.